app/celery/process_pinpoint_receipt_tasks.py

```diff
@@ -122,15 +123,17 @@ def process_pinpoint_results(
             update_notification_status_by_id(notification_id=notification.id, status=notification_status)
 
         current_app.logger.info(
-            'Pinpoint callback return status of %s for notification: %s', notification_status, notification.id
+            'Pinpoint callback return status of %s for notification: %s',
+            notification_status,
+            notification.id,
         )
 
-        statsd_client.incr(f'callback.pinpoint.{notification_status}')
-
-        if notification.sent_at:
-            statsd_client.timing_with_dates(
-                'callback.pinpoint.elapsed-time', datetime.datetime.utcnow(), notification.sent_at
-            )
+        log_notification_total_time(
+            notification.id,
+            notification.created_at,
+            notification_status,
+            'pinpoint',
```

Comment:
Should `pinpoint` be a constant?

Reply:
Yeah, we have 3 of these files. This is the pinpoint file. The "delivery status" file was supposed to eventually include pinpoint but we have not set aside the time to update it and get rid of this file.